gcn_classic_to_json/notices/CALET_GBM_FLT_LC/__init__.py

```diff
@@ -0,0 +1,41 @@
+from ... import utils
+import numpy as np
+
+# Will have to rename schema after the location is decided
+
+
+def parse(bin_arr):
+    # e_high and e_low are both encoded in the same 4 byte integer (as 2 byte shorts)
+    energy_range_data = np.asarray(bin_arr[17:18].data, dtype=">i4").view(">i2")
+    # similarly we do this with lat and long
+    lat_lon_data = np.asarray(bin_arr[16:17].data, dtype=">i4").view(">i2")
+    lat, lon = lat_lon_data[0], lat_lon_data[1]
+    # Using np.unpackbits to get detector status from bits
+    # I have defined 0 in the binary as "on" and 1 as "triggered"
+    detector_options = ["on", "triggered"]
+    # First define as an array with a single 4 byte integer value and then convert to an array of 4 1 byte integer values
+    detectors_bits = np.unpackbits(
+        np.asarray(bin_arr[19:20], dtype=">i4").view(dtype=">u1")
+    )[-3:]
+    detectors_status = [detector_options[bit] for bit in detectors_bits]
+    detectors = dict(zip(["SGM", "HXM1", "HMX2"], detectors_status))
+
+    return {
+        # "$schema" : "",
+        "mission": "CALET",
+        "instrument": "GBM",
+        # "alert_type" : "initial", # does not exist in the binaries
+        # "record_number" = 1, # does not exist in binaries
+        "id": [int(bin_arr[4])],
```

Comment:
Why is it necessary to convert to an int? This field is already an integer.

Reply:
If I don't do an int conversion, I get the following error:
TypeError: Object of type int32 is not JSON serializable

Reply:
https://github.com/nasa-gcn/gcn-classic-to-json/pull/22 will fix that.